lib/middleware/anonymous_cache.rb

```diff
@@ -186,6 +187,10 @@ def key_compress_anon
         GlobalSetting.compress_anon_cache
       end
 
+      def key_show_original_content
+        @request.cookies.key?(ContentLocalization::SHOW_ORIGINAL_COOKIE)
```

Comment:
How and when is the cookie set?

Reply:
What is unoriginal content within this context? Is it content that has been translated?